codes/quantum/qubits/quantum_reed_solomon.yml

```diff
@@ -0,0 +1,44 @@
+#######################################################
```

Comment:
No need for this file; I've created two stub entries for mod-integer and galois-qudit poly codes.